### DISCUSSION TITLE
Running into errors creating a issue even when the calculations are completed

### DISCUSSION THREAD

**Discussion by impiyush** [AlexSim93/pull-request-analytics-action#31] [Q&A] [answered]
Hello,

First of all, thanks a lot for this amazing tool. I am trying to enable this in a private repo for our enterprise and ran into some errors. Hoping you can provide some guidance.
As you can see in the error message, the data was successfully retrieved and the calculations completed, but then it fails in creating the issue. I am assuming there is some particular format it expects for the GITHUB_REPO_FOR_ISSUE and GITHUB_OWNERS_REPOS parameters. You can get an idea of how I am setting it below (I have removed any personally identifiable values)

Side note, I found the parameter docs to be a little confusing and wasn't sure many times what/how to add. Maybe it could use some updating with example column? Also the documentation mentions there is only 1 required parameter, i.e., the token,  but seems like there are more now since you have some validations on them. 

### Error
Data successfully retrieved. Starting report calculations.
Calculation complete. Generating markdown.
/home/runner/work/_actions/AlexSim93/pull-request-analytics-action/v2/build/index.js:11882
      const error = new import_request_error.RequestError(toErrorMessage(data), status, {
                    ^

RequestError [HttpError]: Not Found
    at /home/runner/work/_actions/AlexSim93/pull-request-analytics-action/v2/build/index.js:11882:21
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async requestWithGraphqlErrorHandling (/home/runner/work/_actions/AlexSim93/pull-request-analytics-action/v2/build/index.js:11353:20)
    at async Job.doExecute (/home/runner/work/_actions/AlexSim93/pull-request-analytics-action/v2/build/index.js:13635:18) {
  status: 404,
  response: {
    url: 'https://api.github.com/repos/<GITHUB_OWNER_FOR_ISSUE>/<GITHUB_REPO_FOR_ISSUE>/issues',
    status: 404,
    ...

### Parameters Used
GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
GITHUB_REPO_FOR_ISSUE: "<internal_github_org_name>/<internal_repo_name>"
GITHUB_OWNERS_REPOS: "<internal_github_org_name>/<internal_repo_name>"
GITHUB_OWNER_FOR_ISSUE: "<internal_github_user_name>"
AGGREGATE_VALUE_METHODS: "average"
TIMEZONE: "America/Los_Angeles"
REPORT_DATE_START: ${{ inputs.report_date_start }}
REPORT_DATE_END: ${{ inputs.report_date_end }}
ALLOW_ANALYTICS: "false"

### What else I tried
When I tried with changing the EXECUTION_OUTCOME parameter to "markdown" (or "collection"), I don't see the error but also don't see a markdown or json returned. The action status is passed in this case.

**Comment by AlexSim93** [comment, marked as answer]
Hello, there is a mistake in `GITHUB_REPO_FOR_ISSUE`. You need to set only repository name(internal_repo_name in your case). I think It will fix this error.

Several parameters are optional. For example, `GITHUB_OWNERS_REPOS` is not required, cause you can use `ORGANIZATIONS` input instead of it. So, I decided to not make it required. I will update documentation to make it clearer.

`EXECUTION_OUTCOME` allows you to get markdown or collection as an output. You will have access to this data in next steps in your workflow. It might be useful in case you don't want to use issues for reports.

**Reply by impiyush** [reply]
Thanks for the quick response. I tried updating the parameters these 2 ways

1. with `GITHUB_REPO_FOR_ISSUE` = "<internal_repo_name>", and everything else as it is.

2. with `ORGANIZATIONS` while removing the `GITHUB_OWNERS_REPOS`
`GITHUB_REPO_FOR_ISSUE` = "<internal_repo_name>"
`ORGANIZATIONS` = "<internal_github_org_name>"


In both these cases, I ran into the exact same error as the original post. 

My hypothesis, based on the url in the error message is that it is trying to find the repository under the `GITHUB_OWNER_FOR_ISSUE` which it is unable to find. The repository is under the org, so it should be looking for it there. Thoughts?


For now, I am accessing the markdown and displaying it in the next step in the workflow. Thank you for confirming that.

**Reply by AlexSim93** [reply]
Probably I wasn't clear previous time. `GITHUB_OWNER_FOR_ISSUE` and `GITHUB_REPO_FOR_ISSUE` are required if you want to create/update issue.

**Reply by impiyush** [reply]
but that's what I am trying to say - even when I pass both of those, with the correct values, the error happens. I confirmed it again.

**Reply by AlexSim93** [reply]
Could you share input parameters?
